n6001/hardware/ofs_n6001/build/scripts/run.sh

```diff
@@ -185,11 +138,14 @@ if [ -f "${BSP_FLOW}.failing_paths.rpt" ]; then
 fi
 
 if [ ! -f fpga.bin ]; then
-    echo "ERROR: no fpga.bin found.  FPGA compilation failed!"
+    echo "run.sh ERROR: no fpga.bin found.  FPGA compilation failed!"
     exit 1
 fi
 
-#copy fpga.bin to parent directory so aoc flow can find it
+echo "run.sh: generate acl_quartus_report.txt"
+quartus_sh -t scripts/gen-asp-quartus-report.tcl ofs_top "${BSP_FLOW}"
```

Comment:
the acl-quartus-report used to be generated by adjust-plls.tcl, but that script has now shrunk and been renamed to gen-asp-quartus-report.tcl.